src/inmanta/ast/type.py

```diff
@@ -79,23 +80,20 @@ def get_resolver_for(self, namespace: Namespace):
 
 class Type(Locatable):
     """
-        This class is the base class for all types that represent basic data.
-        These are types that are not relations.
-
-        An instance of Type (or a subclass) represents an Inmanta type.
-        Inmanta instances are represented by instances of other Python types.
+        This class is the abstract base class for all types in the Inmanta :term:`DSL` that represent basic data. These are
+        types that are not relations. Instances of subclasses represent a type in the Inmanta language.
     """
 
     def validate(self, value: Optional[object]) -> bool:
         """
-            Validate the given value to check if it satisfies the constraints
-            associated with this type
+            Validate the given value to check if it satisfies the constraints associated with this type. Returns true iff
```

Comment:
```suggestion
            Validate the given value to check if it satisfies the constraints associated with this type. Returns true if
```

Reply:
This is not a typo. Iff stands for "if and only if"